### ISSUE TITLE
[Episode 04] text mismatch

### ISSUE THREAD

**Issue by laurafortunato** [swcarpentry/git-novice#365]
The output of the command:

`git log --oneline`

is reported as:

```
* 005937f Thoughts about the climate
* 34961b1 Concerns about Mars's moons on my furry friend
* f22b25e Starting to think about Mars
```
But the text does not match the commit messages the student will have typed up to that point (possibly reflecting a previous version?). This may throw off novice users.

The corresponding commit messages the student will have typed are instead:

- Discuss concerns about Mars' climate for Mummy
- Add concerns about effects of Mars' moons on Wolfram
- Start notes on Mars as a base

**Comment by laurafortunato** [comment]
The same applies to the output of the command:

`git log --oneline --graph --all --decorate`

**Comment by dlstrong** [comment]
Hi! I made a pull request based on this. I'm not sure who can accept the pull requests but I don't think I can.

**Comment by iglpdc** [comment]
This was closed by #354. See the comments in #376. Thanks both @laurafortunato and @dlstrong.
